### ISSUE TITLE
"Oops! Something went wrong" When Saving 'Invite User' Entry

### ISSUE THREAD

**Issue by ManZosh** [akaunting/akaunting#2957]
### Akaunting version

3.0.12

### PHP version

8.2.5

### Operating system

Rocky Linux v9.1 Linux v5.14.0-162.18.1.el9_1.x86_64

### Steps to reproduce

1. Fresh installation of Akaunting v3.0.12.
2. All we did was create our basic Company,
3. input Google SMTP config with dedicated email account,
4. Navigate to Profile > Users, click Invite User in top-right
5. try to create an additional 2nd User in the system,
   - inputting a full **Name**,
   - email@company.com business **Email** address
   - Assign to the only **Company** that was just created
   - Give **Role** of either Manager or Accountant
   - Provide any **Landing Page**, I do Dashboard
   - **Language** defaults to US English
7. Click **Save**

### Expected result

A new user should be created and presumably an invitation email dispatched.

### Actual result

Upon completing the web form and clicking Save:
The page loads for a brief moment followed by what appears to be a very brief page refresh.
After about a 2-seconds of page load and refresh, an "**_Oops! Something went wrong_**" message is displayed on the top-right corner of the page which disappears on mouse hover.

### Additional comments

I checked all the log files I could think of... but NOTHING is output anywhere regarding this...
- No errors received in the browser dev console.
- /var/log/nginx/akaunting.error
- /var/log/mariadb/*
- /var/log/php-fpm/*
- /var/www/akaunting/storage/logs/*

...and there's nothing meaningful or related to this issue in any of these log locations!

Are there any other log files I've missed? I checked /var/www/akaunting/config/logging.php but didn't see anything non-standard, app log entries thould output to either nginx's app log or akaunting's ../storage/logs directory.

**System configuration:**
- Rocky Linux v9.1 - SELinux is off, firewalld is off
- nginx v1.20.1
- PHP v.8.2.5 w/ Zend Engine v4.2.5
- MariaDB Server v10.5.16

[I also created a post on the Akaunting forum.](https://akaunting.com/forum/discussion/general/oops-something-went-wrong-when-saving-invite-user-entry)

**Comment by cuneytsenturk** [comment]
@ManZosh Tahank for feedback

I've tested again invite user, and it works. Already our user use to on cloud.

Please  change email service protocol `Log Emails` then test again invite user. if this method works. You should checking STMP configuration.

Also, I'd like to close this issue please continue posting on the form.

**Comment by ManZosh** [comment]
Hi, we ended up totally reinstalling the Akaunting web app on the same VM using the same supporting tech stack and it just ended up working this time around, no more of the issue preventing us from inviting new users any longer. Our SMTP config was the same as well, leveraging smtp.gmail.com:587 over TLS with our dedicated email account. We did have a new issue involving a character in the database user's password that the system didn't like, so maybe that had something to do with the prior issue.

Anyway, unfortunately we couldn't point to a root cause, but a reinstall did the trick!
